### PR TITLE
README.md

### PR DESCRIPTION
**Pull request by VIJAY07102**
Readme document for binary insertion sort

Thanks!

### CODE REVIEW

File: Binary_Insertion_Sort/README.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
+# BINARY INSERTION SORT:
```
Comment (asha15) [inline Binary_Insertion_Sort/README.md]: remove the `:` after the heading

Comment (VIJAY07102) [reply Binary_Insertion_Sort/README.md]: Please review the updated one

**Comment by somya-kapoor** [comment]
Are you working on it?

**Comment by somya-kapoor** [comment]
Due to no activity for more than 7 days, I am closing this PR.

**Pull request closed without merging**